inc/toolbox.class.php

```diff
@@ -2750,10 +2770,10 @@ static function convertImageToTag($content_html, $force_update = false) {
     * @since 9.2
     *
     * @param string $content   html content of input
-    * @param array $tags       list of tags to clen
+    * @param array $tags       list of tags to clean
     *
-    * @return html content
-   **/
+    * @return array|mixed|string
```

Comment:
It will always return string if correctly used passing a strig for first parameter.

Reply:
same as above, also Html::entity_decode_deep could return an array or string.